android/engine/src/main/java/org/smartregister/fhircore/engine/task/FhirTaskPlanWorker.kt

```diff
@@ -96,7 +96,7 @@ constructor(
   }
 
   private fun CarePlan.isLastTask(task: Task) =
-    this.activity.last()?.outcomeReference?.last()?.extractId() == task.logicalId
+    this.activity.lastOrNull()?.outcomeReference?.lastOrNull()?.extractId() == task.logicalId
```

Comment:
this is an existing error, this will throw exceptions with a malformed CarePlan (unclear how likely it is to encounter the malformedness)